src/components/Sing/menuBarData.ts

```diff
@@ -60,12 +78,35 @@ export const useMenuBarData = () => {
     { type: "separator" },
     {
       type: "button",
-      label: "インポート",
+      label: "プロジェクトをインポート",
       onClick: () => {
         void importExternalSongProject();
       },
       disableWhenUiLocked: true,
     },
+    {
+      type: "root",
+      label: "プロジェクトをエクスポート",
+      subMenu: (
+        [
+          ["smf", "MIDI (SMF)"],
+          ["musicxml", "MusicXML"],
+          ["ufdata", "Utaformatix"],
+          ["ust", "UTAU"],
+        ] satisfies [fileType: ExportSongProjectFileType, name: string][]
```

Comment:
細かいけど気づいてしまったので
```suggestion
        ] satisfies [fileType: ExportSongProjectFileType, label: string][]
```
（下の引数名がlabel）